contracts/DepositManager.sol

```diff
@@ -75,53 +73,21 @@ contract DepositManager {
             nameRegistry.getContractDetails(ParamManager.TOKEN_REGISTRY())
         );
         logger = Logger(nameRegistry.getContractDetails(ParamManager.LOGGER()));
-        accountsTree = IncrementalTree(
-            nameRegistry.getContractDetails(ParamManager.ACCOUNTS_TREE())
-        );
-
-        AddCoordinatorLeaves();
-    }
-
-    function AddCoordinatorLeaves() internal {
-        // first 2 leaves belong to coordinator
-        accountsTree.appendLeaf(ZERO_BYTES32);
-        accountsTree.appendLeaf(ZERO_BYTES32);
-    }
-
-    /**
-     * @notice Adds a deposit for the msg.sender to the deposit queue
-     * @param _amount Number of tokens that user wants to deposit
-     * @param _tokenType Type of token user is depositing
-     */
-    function deposit(
-        uint256 _amount,
-        uint256 _tokenType,
-        bytes memory _pubkey
-    ) public {
-        depositFor(msg.sender, _amount, _tokenType, _pubkey);
     }
 
     /**
      * @notice Adds a deposit for an address to the deposit queue
-     * @param _destination Address for which we are depositing
      * @param _amount Number of tokens that user wants to deposit
      * @param _tokenType Type of token user is depositing
      */
     function depositFor(
-        address _destination,
+        uint256 accountID,
```

Comment:
How do I get an account ID to send here?

Reply:
Depositor has to be registered to BLSAccountRegistery in advance